src/Procedures/UpdateTask.sql

```diff
@@ -0,0 +1,92 @@
+
+create or alter proc scheduler.UpdateTask  
+    @taskId            int,
+    @jobIdentifier     sysname,
+    @tsqlCommand       nvarchar(max),
+    @startTime         time,
+    @frequencyType     tinyint,
+    @frequencyInterval smallint,
+    @notifyOperator    sysname,
+    @isNotifyOnFailure bit = 1,
+    @IsEnabled         bit = 0,
+    --@IsCachedRoleCheck bit = 1,

```

Comment:
Why leave in commented out? If its for testing I think it makes sense otherwise not worth checking in deleted code.

Reply:
Nope, just a stub that didn't make the cut when testing in both AG and SI deployment modes. This actually leads to a thought that's been negging me... maybe include `IsCachedRoleCheck` as a vestigial column in the SI deployment to simplify deployment & task administration between installation types? It's already "_functionally always_ `TRUE`" in AG mode. half formed thought but... maybe...? :sweat_smile: